src/main/java/Bron.java

```diff
@@ -0,0 +1,54 @@
+import java.util.Scanner;
+
+public class Bron {
+    public static void main(String[] args) {
+        String logo = "  ____    ____      ____    _     _\n"
+                    + " |  _ \\  |   _\\   /  __  \\ | \\   | |\n"
+                    + " | |_) | | |_) |  | |  | | |  \\  | | \n"
+                    + " |  _ <  | ___/   | |  | | | |\\\\ | |\n"
+                    + " | |_) | | | \\ \\  | |__| | | | \\\\| |\n"
+                    + " |____/  |_|  \\_\\  \\____/  |_|  \\__|\n";
```

Comment:
Should this be a static variable?

Reply:
Should `logo` be capitalised as `LOGO` instead to follow the naming convention for constants?